wnfs/benches/namefilter.rs

```diff
@@ -0,0 +1,74 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion, Throughput};
+use wnfs::{dagcbor, private::namefilter::Namefilter};
+
+fn add(c: &mut Criterion) {
+    c.bench_function("namefilter add", |b| {
+        b.iter(|| {
+            let mut namefilter = black_box(Namefilter::default());
```

Comment:
Same thing here. We could/should split the initialization out with `iter_batched`.